### PR TITLE
CSS label format

### PR DESCRIPTION
**Pull request by mikeknep**

### CODE REVIEW

File: src/gretel_trainer/relational/report/report_template.html [hunk 1]
```diff
@@ -58,7 +58,7 @@ <h1>Gretel Relational<br />Synthetic Report</h1>
       <h2>Table Relationships</h2>
 
       <p>
-        The primary and foreign keys for each table in the synthesized database and their
```
Comment (mikeknep) [inline src/gretel_trainer/relational/report/report_template.html:61]: Just a note: I confirmed that deleting these trailing whitespaces at the ends of lines does **not** cause words to be jammed together in the final output

File: src/gretel_trainer/relational/report/report_template.html [hunk 2]
```diff
@@ -111,21 +111,23 @@ <h2>Synthetic Data Quality Results</h2>
...
           {% for table, evaluation in presenter.evaluations.items() %}
+          {% set individual_grade = presenter.sqs_score_to_grade(evaluation.individual_sqs) %}
+          {% set cross_table_grade = presenter.sqs_score_to_grade(evaluation.cross_table_sqs) %}
           <tr>
             <td><b>{{ table }}</b></td>
             <td>
               <span class="sqs-table-score">{{ evaluation.individual_sqs }}</span>
-              <span class="label label__{{ presenter.sqs_score_to_grade(evaluation.individual_sqs).lower().replace(' ', '-') }}">
-                {{ presenter.sqs_score_to_grade(evaluation.individual_sqs) }}
+              <span class="label label__{{ presenter.css_label_format(individual_grade) }}">
```
Comment (tylersbray) [inline src/gretel_trainer/relational/report/report_template.html:120]: I only saw your ask around this after I had merged in that other PR with the js, thanks for doing this.  The `set` above is nice.

File: src/gretel_trainer/relational/report/report_template.html (hunk 1 above)
Comment (tylersbray) [reply src/gretel_trainer/relational/report/report_template.html:61]: I will permit this.

**Review by tylersbray — APPROVED** [review]
Looks good.  This was the long one, right?

**Pull request merged**